### PR TITLE
Added Joseph Bishop to current students lists

### PR DESCRIPTION
**Pull request by joeybish**
Added Joseph Bishop as a current student in the student cohort sections. In both the README.md and index.html file (properly this time).

### CODE REVIEW

**Comment by joeybish** [comment]
Closes #58

File: index.html [hunk 1]
```diff
@@ -129,6 +129,13 @@ <h4>Student Support</h4>
                     </li>
                   </ul>
                 </li>
+                <li><a href="https://github.com/joeybish" target="_blank" rel="noopener noreferrer">Joseph Bishop</a>
+                  <ul>
+                    <li>
+                      Majors: Ecology & Evolutionary Biology and Optical Sciences & Engineering
```
Comment (reyg3) [inline index.html:135]: @tweissaz do we care about "&" vs `&amp;`? I'm not sure what is best practice these days. I also see "&" in other parts of the index.html file.

**Review by reyg3 — APPROVED** [review]
Additions look good and follow conventions.

File: index.html (hunk 1 above)
Comment (tweissaz) [reply index.html:135]: I think this should be fine, as we're using it within the rest of the document and it's a pretty standard character at this point.

**Review by tweissaz — APPROVED** [review]
These additions look good to me, and it seems you addressed everything I mentioned previously. Should be good to merge!

**Comment by tweissaz** [comment]
Going to go ahead and merge this!

**Pull request merged**